### ISSUE TITLE
Similarity learning reference code is not documented

### ISSUE THREAD

**Issue by Naruto-Sasuke** [pytorch/vision#1138]
@dakshjotwani @fmassa

**Comment by fmassa** [comment]
It's not documented yet indeed, but there are still some more things that should be done to improve it.

I'm thinking for example of having a larger-scale dataset for similarity learning, like faces, and modify the example to use this dataset instead.

Then, I will look into putting it in the website.

**Comment by dakshjotwani** [comment]
I agree that it would be better to have a large scale face dataset like VGGFace2 to demonstrate the use of the reference code. I was planning on working on it next week.

I shall open an issue to discuss the inclusion of VGGface2 and keep you posted @Naruto-Sasuke. In the meantime, maybe I can help you with any issues that you might be facing?
